src/views/List.jsx

```diff
@@ -116,31 +117,43 @@ export function List({ data, listId }) {
 
 		return (
 			<>
-				<form>
-					<label htmlFor="filter">Filter List</label>
-					<br />
-					<input
-						type="text"
-						name="filter"
-						id="filter"
-						value={searchInput}
-						placeholder="Item name"
-						onChange={(e) => {
-							setSearchInput(e.target.value);
-						}}
-						onKeyDown={(e) => {
-							handleKeyDown(e);
-						}}
-					></input>
-					<button
-						type="button"
-						name="clearInput"
-						onClick={(e) => {
-							resetDisplayList(e);
-						}}
-					>
-						X
-					</button>
+				<form className="pb-5">
+					<div className="flex flex-col justify-center items-center">
+						<label htmlFor="filter" className="px-3 mt-2">
+							Filter List:
+						</label>
+						<div className="w-full sm:w-1/2 flex items-center border-2 rounded-lg py-2 px-3 sm:px-5">
+							<FontAwesomeIcon
+								icon={pen}
+								title="Enter item name"
+								className="text-gray-500 mr-2 sm:mr-4"
+							/>
+							<input
+								className="flex-grow border-none outline-none bg-transparent"
+								type="text"
```

Comment:
the background for dark mode should be the light blue in both this and the add item input.